cmake/compiler/gcc/target.cmake

```diff
@@ -13,6 +13,7 @@ find_program(CMAKE_AR         ${CROSS_COMPILE}ar      PATH ${TOOLCHAIN_HOME} NO_
 find_program(CMAKE_RANLIB     ${CROSS_COMPILE}ranlib  PATH ${TOOLCHAIN_HOME} NO_DEFAULT_PATH)
 find_program(CMAKE_READELF    ${CROSS_COMPILE}readelf PATH ${TOOLCHAIN_HOME} NO_DEFAULT_PATH)
 find_program(CMAKE_GDB        ${CROSS_COMPILE}gdb     PATH ${TOOLCHAIN_HOME} NO_DEFAULT_PATH)
+find_program(CMAKE_GDB        gdb-multiarch           PATH ${TOOLCHAIN_HOME}                )
```

Comment:
CMAKE_GDB is seemingly found twice: This could lead one to believe the second `find_program` would always win. 

CMake docs say:

>  If the program is found the result is stored in the variable and the search will not be repeated unless the variable is cleared.

... it would be nice to add a small comment about this.